lib/puppet/functions/binary_file.rb

```diff
@@ -12,7 +12,7 @@
   def binary_file(scope, unresolved_path)
     path = Puppet::Parser::Files.find_file(unresolved_path, scope.compiler.environment)
     unless path && Puppet::FileSystem.exist?(path)
-      raise Puppet::ParseError, _("binary_file(): The given file '%{unresolved_path}' does not exist") % { unresolved_path: unresolved_path }
+      raise Puppet::ParseError, _("The given file '%{unresolved_path}' does not exist") % { unresolved_path: unresolved_path }
```

Comment:
The original message included that this was coming from usage of the `binary_file()` function. Does this still include enough context for people to know that without prefixing the message with `binary_file(): `?

Reply:
I removed the "binary_file():" method name on purpose, It doesn't provide value to the user, and we'd have to tell the translators to not translate it.